tests/runtime/engine/runner.py

```diff
@@ -307,7 +307,7 @@ def run_test(test):
 
         if test.cleanup:
             try:
-                cleanup = subprocess.run(test.cleanup, shell=True, capture_output=True, text=True)
+                cleanup = subprocess.run(test.cleanup, shell=True, stderr=subprocess.PIPE)
```

Comment:
All the other callsites use `universal_newlines=True` which is equivalent to `text=True`. So maybe set that too?

Also how about capturing both stdout and stderr? I see below we only print stderr, but buffering both stdout & stderr would help reduce noise for successful CLEANUPs